support-frontend/assets/components/subscriptionCheckouts/addressSearch/__tests__/addressFieldsStore.ts

```diff
@@ -1,178 +0,0 @@
-// ----- Imports ----- //
```

Comment:
These are duplicates of the tests under `support-frontend/assets/components/subscriptionCheckouts/address/__tests__/addressFieldsStore.ts`